datastore/shared/postgresql_backend/sql_read_database_backend_service.py

```diff
@@ -312,7 +313,27 @@ def get_deleted_status_from_events(
         result = self.connection.query(query, [tuple(fqids)])
         return {row["fqid"]: row["type"] == EVENT_TYPES.DELETE for row in result}
 
-    def get_max_position(self) -> int:
+    def get_history_information(
+        self, fqids: List[Fqid]
+    ) -> Dict[Fqid, List[HistoryInformation]]:
+        positions = self.connection.query(
+            """select fqid, position, timestamp, user_id, information from positions natural join events
+            where fqid in %s and information::text<>%s::text order by position asc""",
```

Comment:
Why is the check for the non-empty information necessary? Don't we provide information for every position?

Reply:
In the UI we only want to include positions, that have useful information; this is at least the plan. The backend should only write information, if it is interesting for the user. What it is is mostly decided by Emanuel at some point. So the backend only writes "necessary" information and the DS filters out "unnecessary" ones (Since why should it even provide unnecessary information;) )